src/index.js

```diff
@@ -90,7 +90,7 @@ const run = async () => {
 
                 // Validate if branch contains breaking changes
                 // and version has the same major version as previous.
-                if (branch === 'release') {
+                if (branch.startsWith('release')) {
```

Comment:
Is that only for `release` because we don't expect it to happen on `hotfix` or am I'm missing something?

Reply:
Yes, the way I understood it major version bump should occur only on release, hotfix should not be major but modify patch version of semver.